src/content/docs/logs/forward-logs/overview of log-forwarding.mdx

```diff
@@ -0,0 +1,122 @@
+
+We offer many options to forward host logs, which are listed in the table below. Basically, you can use one of our log fowarding options (i.e., our infrastructure agent, Kubernetes plugin, or log API) or a third-party log forwarder (i.e., AWS, Azure, GCP).
+
+To get started with log forwarding, click one of the links in the table and follow instructions on that page. 
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "250px" }}>
+        If you want to send logs from...
+      </th>
+      <th>
+        Use one of these log forwarders
+      </th>
+    </tr>
+  </thead>
+
+    <tbody>
+    <tr>
+      <td>
+        Operating systems
+      </td>
+      <td>
+        * (Recommended) Enable our infrastructure agent to send your host logs. By default, our APM agents send application-side logs, but if you want to also send host logs, you can do things like collect custom attributes, which you can't do with APM agents. To get started, see [Configure the infrastructure agent to forward logs](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/).
+        * [Fluent Bit plugin](/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding)
+        * [Fluentd plugin](/docs/logs/forward-logs/fluentd-plugin-log-forwarding)
+        * [Logstash plugin](/docs/logs/forward-logs/logstash-plugin-log-forwarding/)
+        * [Syslog/TCP](/docs/logs/log-management/log-api/use-tcp-endpoint-forward-logs-new-relic)
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Cloud environments
+      </td>
+      <td>
+       * Amazon AWS:
+        * [CloudFront plugin](/docs/logs/forward-logs/cloudfront-web-logs)
+        * [CloudWatch plugin](/docs/logs/forward-logs/aws-lambda-sending-cloudwatch-logs/)
+        * [FireLens plugin](/docs/logs/forward-logs/aws-firelens-plugin-log-forwarding/)
+        * [Kinesis Firehose](/docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose/)
+        * [AWS Lambda for sending logs from S3](/docs/logs/forward-logs/aws-lambda-sending-logs-s3/)
+       * [Google Cloud platform](/docs/logs/forward-logs/google-cloud-platform-log-forwarding)
+       * [Microsoft Azure](/docs/logs/forward-logs/azure-log-forwarding/)
+       * [Heroku](/docs/logs/forward-logs/heroku-log-forwarding/)
+       * [Cloudflare](/docs/logs/forward-logs/cloudflare-logpush-forwarding)
+      </td>
+    </tr>
+    
+    <tr>
+      <td>
+        Docker/Kubernetes containers
+      </td>
+      <td>
+        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
```

Comment:
@lchapman4 @marcelschlapfer I want to move these three bullet points over to the "Forward logs with infra agent" page, as a new section that can be linked to from this table. But for this first bullet point, does this mean APM logs in context automatically collects Docker logs? Or does it mean in conjunction with the infra agent?